kornia/augmentation/container/augment.py

```diff
@@ -69,10 +69,10 @@ class AugmentationSequential(ImageSequential):
         >>> aug_list = AugmentationSequential(
         ...     kornia.augmentation.ColorJitter(0.1, 0.1, 0.1, 0.1, p=1.0),
         ...     kornia.augmentation.RandomAffine(360, p=1.0),
-        ... data_keys=["input", "mask", "bbox", "keypoints"],
-        ... return_transform=False,
-        ... same_on_batch=False,
-        ... random_apply=10,
+        ...     data_keys=["input", "mask", "bbox", "keypoints"],
+        ...     return_transform=False,
+        ...     same_on_batch=False,
+        ...     random_apply=10,
```

Comment:
Are you okay to add few lines of description and  code to demonstrate the different behavior of ```random_apply``` as requested from [here](https://github.com/kornia/kornia/issues/1159)?

Reply:
Hi @shijianjian, I'm not sure I fully understand what `random_apply` does. I can look into it and add it in though, just need some time if that's fine with you guys.

Reply:
@shijianjian isn't this out of the scope of this PR ?